src/TcpAdapterProxy.cpp

```diff
@@ -2090,32 +2112,14 @@ namespace aws { namespace iot { namespace securedtunneling {
                     else
                     {
                         BOOST_LOG_SEV(log, debug) << "Resolved bind IP: " << results->endpoint().address().to_string();
-                        boost::system::error_code bind_ec;
 
-                        client->connectionId_to_tcp_connection_map[connection_id]->socket().open(results->endpoint().protocol());
-                        client->connectionId_to_tcp_connection_map[connection_id]->socket().bind({results->endpoint().address(), 0}, bind_ec);
```

Comment:
Can you explain why we're removing these logic on IP binding

Reply:
calling bind() does not make sense on the destination side because we are going to call connect() to a port already in use by a server process

https://stackoverflow.com/questions/27014955/socket-connect-vs-bind